www/language.js

```diff
@@ -144,6 +144,10 @@ module.exports = {
     The minimum number of search characters needed before filtering begins.
   `,
 
+  minSearch: d => stripIndent`
+    The minimum number of search characters needed before the list appears.
+  `,
```

Comment:
I think since this is only relevant to Multiselect you can do it as a jsDoc-esque comment on the prop-type in the file